CHANGELOG.md

```diff
@@ -5,19 +5,26 @@
 A breaking change should be clearly marked in this log.
 
 
-## Unreleased
+## v1.0.0-beta.3
```

Comment:
should we include expiration data #153  on this release?

Reply:
yep, just needed upstream merge